angular/src/app/components/comment/comment.component.ts

```diff
@@ -9,8 +9,14 @@ import {PostComment} from "../../interfaces/post-comment";
 export class CommentComponent implements OnInit {
 
   @Input() comment?: PostComment
+  date?: string
```

Comment:
delete this one line too(don't forget to remove this form html as well) :cactus: